documentation/docs/developer_guide/coding_standards.md

```diff
@@ -1 +1,178 @@
-# Coding standards
\ No newline at end of file
+
+## Introduction
+
+CABLE is a community software project. 
+For the benefit of the whole community, developers are required to adhere to certain coding standards that better enable portability, efficient maintenance, future development, performance and readability. 
+
+Submitted developments that are inconsistent with the standards discussed below, and/or poorly commented/documented code will not be accepted without revision. 
+
+The JULES and CABLE (JAC) project is a long term initiative to include CABLE as an inter-operable Land-Surface Model (LSM) within the Unified Model (UM). For several reasons, CABLE is coupled to the UM via its native LSM, JULES. From a coding standards perspective, both the UM and JULES developments are strictly goverened by the UK Met Office and thus CABLE's coding standards are somewhat constrained the [JULES coding standards](http://jules-lsm.github.io/coding_standards/).
+
+Currently not all of the CABLE code is compliant with the standards described here. Work is in progress to apply these standards across the existing code base, however these standards are expected to be met by any new development.
+
+Whilst syntax/text formatting style is relatively trivial, it must still be adhered to. Some more significant general rules follow.
+
+## General Rules
+
+### 1. IO must be restricted to the top level
+
+In terms of the offline application of the model this means IO should be restricted to the offline/ directory. There should be no IO code elsewhere in the CABLE directory. 
```

Comment:
```suggestion
In terms of the offline application of the model this means IO should be restricted to the `offline/` directory. There should be no IO code elsewhere in the CABLE directory. 
```
otherwise it looks like offline/directory